apax/nodes/model.py

```diff
@@ -204,9 +219,96 @@ def get_calculator(self, **kwargs) -> ase.calculators.calculator.Calculator:
             for transform, params in self.transformations.items():
                 transformations.append(available_transformations[transform](**params))
 
-        calc = ASECalculator(
-            model_dir,
-            dr=self.nl_skin,
-            transformations=transformations,
+        with self.state.use_tmp_path():
+            calc = ASECalculator(
+                model_dir,
+                dr=self.nl_skin,
+                transformations=transformations,
+            )
+            return calc
+
+
+class ApaxCalibrate(ApaxBase):
+    """Globally calibrate the energy and force uncertainties of an Apax model.
+
+    Parameters
+    ----------
+    model: Apax
+        Model to calibrate
+    validation_data: list[ase.Atoms]
+        Calibration atoms
+    batch_size: int, default = 32
+        Processing batch size. Choose the largest allowed by your VRAM.
+    criterion: str, default = "ma_cal
+        Calibration criterion. See uncertainty_toolbox for more details.
+    shared_factor: bool, default = False
+        Whether or not to calibrate energies and forces separately.
+    optimizer_bounds: Tuple[float, float], default = (1e-2, 1e2)
+        Search value bounds.
+    transformations: dict
+        Key-parameter dict with function transformations applied
+        to the model function within the ASE calculator.
+        See the apax documentation for available methods.
+    """
+
+    model: t.Any = zntrack.deps()
+    validation_data: list[Atoms] = zntrack.deps()
+    batch_size: int = zntrack.params(32)
+    criterion: str = zntrack.params("ma_cal")
+    shared_factor: bool = zntrack.params(False)
+    optimizer_bounds: t.Tuple[float, float] = zntrack.params((1e-2, 1e2))
+
+    transformations: t.Optional[list[dict[str, dict]]] = zntrack.params(None)
+
+    nl_skin: float = zntrack.params(0.5)
+
+    metrics = zntrack.metrics()
+
+    e_factor: float = zntrack.outs()
+    f_factor: float = zntrack.outs()
```

Comment:
Would it be sufficient to just store them in `metrics`?

Reply:
I guess so